.github/workflows/create-release.yaml

```
@@ -29,13 +33,39 @@ jobs:
               - TBD
           draft: false
           prerelease: false
-  wait-for-template:
-    # this step waits until proper Prow job uploads the template.yaml file
+      - name: Create release branch
+        run: |
+            git checkout -b ${{ github.event.inputs.name }}
+            git push --set-upstream origin ${{ github.event.inputs.name }}
+  wait-for-prow-jobs:
+    # this step waits until proper Prow job uploads the template.yaml file, docker image and module OCI image
     runs-on: ubuntu-latest
+    timeout-minutes: 5
     steps:
-      - name: Wait
+      - name: Wait for template.yaml
         run: |
           until $(curl --output /dev/null --silent --head --fail https://github.com/kyma-project/btp-manager/releases/download/${{ github.event.inputs.name }}/template.yaml); do
             echo 'waiting...'
             sleep 15
           done
+      - name: Wait for btp-operator module
+        # Prow job which uploads the template.yaml also pushes the OCI module image to the registry, so there is no need to make retries for btp-operator module image. It must exist.
+        run: |
+          found=$(skopeo list-tags $BTP_OPERATOR_REPO | jq '.Tags|any(. == "${{ github.event.inputs.name }}")')
+          if [ $found == "false" ]; then
+            echo "Image $BTP_OPERATOR_REPO:${{ github.event.inputs.name }} not found"
+            exit 1
+          else
+            echo "Image found"
+          fi
+      - name: check btp manager
```

You consistently start all step names with uppercase letter. Except here...
```suggestion
      - name: Check btp manager
```